### PR TITLE
[C-3001, C-3003] Fix new upload flow error formatting

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Updating errors to have a period at the end and update the icon to be red

### How Has This Been Tested?
Manually Tested

### CODE REVIEW

**Review by amendelsohn — APPROVED** [review]
🔥

File: packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx [hunk 1]
```diff
@@ -55,7 +55,6 @@ export const MultiTrackSidebar = () => {
                 className={styles.iconError}
                 icon={IconError}
                 size='xSmall'
-                fill='accentRed'
```
Comment (dylanjeffers) [inline packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx:58]: How come this doesn't work? Wondering if we can update the icon component to make it work?

**Review by dylanjeffers — APPROVED** [review]
Looks great just wondering about the icon

File: packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx (hunk 1 above)
Comment (Kyle-Shanks) [reply packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx:58]: not sure. I can take a look though and see if its easy to fix

Comment (dylanjeffers) [reply packages/web/src/pages/upload-page/fields/MultiTrackSidebar.tsx:58]: Awesome yeah lmk

**Pull request merged**